apis/openstack-heat/src/main/java/org/jclouds/openstack/heat/v1/domain/StackResource.java

```diff
@@ -16,235 +16,234 @@
         */
 package org.jclouds.openstack.heat.v1.domain;
 
-import com.google.common.base.CaseFormat;
-import com.google.common.base.Objects;
-import com.google.common.collect.ImmutableSet;
-import org.jclouds.openstack.v2_0.domain.Link;
-import org.jclouds.openstack.v2_0.domain.Resource;
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
please change the code style to be:
Tab size :3
Indent:3
Continuation indent:6